ui/packages/catalog-ui-search/src/main/webapp/component/property/property.less

```diff
@@ -2,7 +2,7 @@
   display: block;
   width: 100%;
   white-space: nowrap;
-  padding: @minimumSpacing;
+  padding: @minimumSpacing 1.5rem;
```

Comment:
Make sure this doesn't affect how the user-settings side panel and inspector view while editing

Reply:
I checked those and they look good